packages/devtools-kit/src/_types/rpc.ts

```diff
@@ -55,8 +55,8 @@ export interface ServerFunctions {
   // Queries
   getImageMeta(filepath: string): Promise<ImageMeta | undefined>
   getTextAssetContent(filepath: string, limit?: number): Promise<string | undefined>
-  writeStaticAssets(token: string, file: { name: string; data: string }[], path: string): Promise<string[]>
-  deleteStaticAsset(token: string, file: string): Promise<void>
+  writeStaticAssets(token: string, file: { path: string; content: string; encoding?: BufferEncoding; update?: boolean }[], folder: string): Promise<string[]>
```

Comment:
```suggestion
  writeStaticAssets(token: string, file: { path: string; content: string; encoding?: BufferEncoding; override?: boolean }[], folder: string): Promise<string[]>
```

Maybe this would be more clear